modules/internal/String.chpl

```diff
@@ -1121,19 +1120,22 @@ module String {
       var localThis: string = this.localize();
 
       var i = 0;
-      while i < localThis.buffLen {
+      try!{
```

Comment:
space before {

Reply:
And I don't like `try!` in such huge blocks. decodeHelp is the only thing that can throw here. And you can have `try!` expressions (as well as statements, which is what you have here, so the call to decodeHelp below should change as:

```chapel
const (cp, nBytes) = try! decodeHelp(blah blah)
```